Task2/usersAPI.ts

```diff
@@ -11,34 +11,28 @@ const port = process.env.PORT || 3000;
 app.use(express.json());
 app.use('/users', router);
 
-function getAutoSuggestUsers (loginSubstringin:string, limit: string):any {
-	if (!loginSubstringin || loginSubstringin.length === 0) return usersCollection;
-	loginSubstringin = loginSubstringin.toLowerCase();
-	const numLimit: number = limit ? +limit : usersCollection.length;
+function getAutoSuggestUsers (loginSubstring: string, limit: number):any {
+	const collectionLimit: number = limit ? limit : usersCollection.length;
 
-	const filteredAndSortedUsersCollection = usersCollection.filter(user => {
-		return user.login.startsWith(loginSubstringin);
-	}).sort((userA, userB) => {
-		return userA.login < userB.login ? -1 : 1;
-	});
-	const limitCollection = Math.min(
-		filteredAndSortedUsersCollection.length, 
-		numLimit);				
-	return filteredAndSortedUsersCollection.slice(0, limitCollection);
-	
+	if (!loginSubstring || loginSubstring.length === 0) return usersCollection.slice(0, collectionLimit);
+	loginSubstring = loginSubstring.toLowerCase();
```

Comment:
Best practice is not to modify parameters.
That's the reason 
![image](https://user-images.githubusercontent.com/17451948/72529717-e8bda980-387e-11ea-8021-46d5456d8ae1.png)